src/Deposit.js

```diff
@@ -509,16 +518,19 @@ export default class Deposit {
     return {}
   }
 
-  async getOwner() /* : Promise<string>*/ /* ETH address */ {
-    return await this.factory.depositTokenContract.methods
-      .ownerOf(this.address)
+  /**
+   * @return {Promise<string>} The ETH address of the deposit owner.
+   */
+  async getOwner() {
+    return await this.factory
+      .depositToken()
+      .methods.ownerOf(this.address)
       .call()
   }
 
   async inVendingMachine() /* : Promise<boolean>*/ {
     return (
-      (await this.getOwner()) ==
-      this.factory.vendingMachineContract.options.address
+      (await this.getOwner()) == this.factory.vendingMachine().options.address
```

Comment:
I know this is from before, but is this intentionally `==` instead of `===`?

Reply:
I don't know that it was intentionally that, though I don't think it hurts. Can bump it to a third `=` though :grin:

Reply:
Fix't.